src/DashboardView/Settings/Settings.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import Modal from '../../Utils/Modal/Modal.js';
+import EventTypesSettings from './EventTypes/EventTypes.js';
+
+class Settings extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            settingsVisible: false,
+        };
+        //function binding
+        this.modalShow = this.modalShow.bind(this);
+        this.onSubmit = this.onSubmit.bind(this);
+    }
+    
+    modalShow() {
+        //do not prevent default!
+        this.setState({settingsVisible: true});
+    }
+    
+    onSubmit() {
+        $('#settings-modal').modal('hide');
+        this.setState({settingsVisible: false});
+    }
+    
+    componentDidMount() {
+        $('#settings-modal').on('hidden.bs.modal', function (event) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "event". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)